plugins/dell/fu-plugin-dell.c

```diff
@@ -603,6 +608,7 @@ fu_plugin_dell_detect_tpm (FuPlugin *plugin, GError **error)
 	fu_device_set_id (dev, tpm_id);
 	fu_device_add_guid (dev, tpm_guid);
 	fu_device_set_name (dev, pretty_tpm_name);
+	fu_device_set_summary (dev, "Primary TPM device");
```

Comment:
This should really just be "Platform TPM device"

Reply:
Done.